code/modules/surgery/organs/organ_external.dm

```diff
@@ -386,16 +386,18 @@ This function completely restores a damaged organ to perfect condition.
 	surgeryize()
 	if(is_robotic())	//Robotic organs stay robotic.
 		status = ORGAN_ROBOT
-	else if(HAS_TRAIT(owner, TRAIT_I_WANT_BRAINS))
+	else if(owner && HAS_TRAIT(owner, TRAIT_I_WANT_BRAINS))
 		status = ORGAN_DEAD
 	else
 		status = 0
 	germ_level = 0
 	perma_injury = 0
 	brute_dam = 0
 	burn_dam = 0
-	if(!HAS_TRAIT(owner, TRAIT_I_WANT_BRAINS)) // zombies's wounds don't close. Because thats cool.
-		open = ORGAN_CLOSED //Closing all wounds.
+	if(owner && !HAS_TRAIT(owner, TRAIT_I_WANT_BRAINS)) // zombies's wounds don't close. Because thats cool.
+		open = ORGAN_CLOSED
+	else
+		open = ORGAN_CLOSED //Closing all wounds for detached limbs.
```

Comment:
You want this, because otherwise you're making it so zombie limbs will close
```suggestion
	if(!owner || !HAS_TRAIT(owner, TRAIT_I_WANT_BRAINS)) // zombies's wounds don't close. Because thats cool.
		open = ORGAN_CLOSED //Closing all wounds for detached limbs.
```

Reply:
![image](https://github.com/user-attachments/assets/afe3b604-39fd-480a-bb4e-524132b8c0a7)
your variant doesnt help zombies.

Reply:
Limbs doesnt have owner, so they runtime on this path and doesnt continue code at all

Reply:
![image](https://github.com/user-attachments/assets/acb1c2f1-91a1-4210-a1c5-bfe5bfd14422)
this code also doesnt work somehow. Why..

Reply:
Don't resolve conversations from a maintainer without them saying it can be closed (if the review isn't applied)
In this case, the `else` statement made every dropped limb closed regardless of the `if` statement, while we want dropped limbs from zombies or with owners to stay open.

Reply:
Zombies cant drop limbs because of their feature? so we need close wounds for all, except zombie limbs on their owner.

For me, better just delete this "feature", this is pointless if compare to mito broken interaction